### ISSUE TITLE
Edit IMPORT TABLE statement on Migrate from MySQL doc

### ISSUE THREAD

**Issue by HeavyHorst** [cockroachdb/docs#8449]
Re: [Migrate from MySQL](https://cockroachlabs.com/docs//v20.1/migrate-from-mysql.html)

## Issue Description

I think the command to Import a table from a full database dump is wrong.

```
CREATE DATABASE IF NOT EXISTS employees;
USE employees;
IMPORT MYSQLDUMP 'https://s3-us-west-1.amazonaws.com/cockroachdb-movr/datasets/employees-db/mysqldump/employees.sql.gz';
```

## Suggested Resolution 

shouldn't that be:
```
CREATE DATABASE IF NOT EXISTS employees;
USE employees;
IMPORT TABLE employees FROM MYSQLDUMP 'https://s3-us-west-1.amazonaws.com/cockroachdb-movr/datasets/employees-db/mysqldump/employees-full.sql.gz';
```

**Comment by lnhsingh** [comment]
Hi @HeavyHorst - yes, you are correct! Thank you for filing this issue.

**Comment by UdokaVrede** [comment]
Hi @HeavyHorst are you fixing this?

**Comment by HeavyHorst** [comment]
@UdokaVrede If you have time you are welcome to do this.

**Comment by Amruta-Ranade** [comment]
@UdokaVrede Let me know if you want me to assign the issue to you :)

**Comment by UdokaVrede** [comment]
Hi @Amruta-Ranade , I would love to fix this

**Comment by Amruta-Ranade** [comment]
Addressed in #8527 
Thanks for your contribution, @UdokaVrede! And thanks for filing the issue, @HeavyHorst!
